src/Analyzers/CSharp/Tests/AddExplicitCast/AddExplicitCastTests.cs

```diff
@@ -3076,6 +3076,34 @@ public void M(object o)
 
         if (array[(int)o] > 0) {}
     }
+}");
+        }
+
+        [Fact, WorkItem(41500, "https://github.com/dotnet/roslyn/issues/41500")]
+        public async Task RemoveExistingCast1()
```

Comment:
This test is almost the same scenario as the skipped test I linked in last comment. I think this could be deleted in favor of unskipping the existing tests.

Reply:
don't care :)  unskipped the otehrs.